packages/admin/src/utils/config.ts

```diff
@@ -0,0 +1,18 @@
+import { Config as ConfigSchema } from '../config';
+import crypto from 'crypto';
+import { isNil } from 'lodash';
+
+export async function generateConfigDefaults(
+  config: ConfigSchema,
+): Promise<ConfigSchema> {
+  const tokenSecretConfig = config.auth.tokenSecret;
+  const hostUrlConfig = config.hostUrl;
+  if (tokenSecretConfig === '' || isNil(tokenSecretConfig)) {
```

Comment:
This is going to regenerate the secret, contrary to Router's case where we merge the incoming config with whatever's already stored in Redis/Database.
Should we make this behavior consistent?